priv/sql/155-yokozuna-tests-update.sql

```diff
@@ -0,0 +1,111 @@
+BEGIN;
+
+-- Update existing tests that were set to 2.1.0 or 2.2.0 to 2.0.6
+UPDATE tests SET min_version_a = '{2,0,6}', max_version_a = '{2,0,99}'
+ WHERE name LIKE 'yz_%' AND (min_version_a = '{2,1,0}' OR min_version_a = '{2,2,0}');
```

Comment:
Just to confirm - you are sure there are no other test entries that match yz_% that would be updated by this, right?